scene/gui/scroll_container.cpp

```diff
@@ -623,10 +632,32 @@ void ScrollContainer::_bind_methods() {
 	BIND_ENUM_CONSTANT(SCROLL_MODE_RESERVE);
 
 	BIND_THEME_ITEM_CUSTOM(Theme::DATA_TYPE_STYLEBOX, ScrollContainer, panel_style, "panel");
+#ifdef TOOLS_ENABLED
+	BIND_THEME_ITEM_CUSTOM(Theme::DATA_TYPE_STYLEBOX, ScrollContainer, focus_style, "focus");
+#endif
 
 	GLOBAL_DEF("gui/common/default_scroll_deadzone", 0);
 };
 
+#ifdef TOOLS_ENABLED
+void ScrollContainer::set_draw_focus_border(bool p_draw) {
+	_draw_focus_border = p_draw;
+}
+
+bool ScrollContainer::get_draw_focus_border() {
+	return _draw_focus_border;
+}
+
+bool ScrollContainer::child_has_focus() {
+	bool it_has_focus = false;
+	const Control *focus_owner = get_viewport() ? get_viewport()->gui_get_focus_owner() : nullptr;
+	if (focus_owner && is_ancestor_of(focus_owner)) {
+		it_has_focus = true;
+	}
+	return it_has_focus;
```

Comment:
You don't need `it_has_focus` variable.

`return focus_owner && is_ancestor_of(focus_owner);`